elastic/kibana
# [Security Solution][Detection Engine] fixes reindexed .list(from 7.x) index migration to DS

## Summary

- addresses https://github.com/elastic/security-team/issues/11414


### Testing

1. Create cloud env of 7.x version
2. Upload list items
3. Create cloud env of 8.18 from existing 7.x snapshot (from previous steps)
4. Connect local Kibana of 8.18 from mirror branch of this one(https://github.com/elastic/kibana/pull/205359)
5. Add to Kibana dev config following options to enable Upgrade assistant(UA) showing outdated indices
    ```yml
    xpack.upgrade_assistant.featureSet:
      mlSnapshots: true
      migrateDataStreams: true
      migrateSystemIndices: true
      reindexCorrectiveActions: true
    ```  
6. When Kibana started DO NOT visit Detection rule or any Security  page
7. Open KIbana Upgrade Assistant, check Elasticsearch deprecations
8. Find outdated .lists-* and .items-* index
9. Migrate them
10. Visit detection page to ensure list indices moved to data stream stream and lists functionality is available: view, upload delete, etc
